src/components/SearchResults.jsx

```diff
@@ -0,0 +1,36 @@
+import { React, useState } from "react";
+import PropTypes from "prop-types";
+import ImageGallery from 'react-image-gallery';
+import "../styles/searchResults.css";
+
+function SearchResults({ results }) {
+  if (!results.length) {
+    return (
+      <div id="results-heading">
+        No results
+      </div>
+    )
+  }  
+  
+  return(
+      <div>
+        <div id="results-heading">{results.length} results</div>
+        <div className="image-results">
+          <ImageGallery items={results} thumbnailPosition="left" showIndex={true} showFullscreenButton={false} lazyLoad={true} />
+        </div>
+      </div>
+    )
+  }
+
+
+SearchResults.defaultProps = {
+  length: 0,
+}
+
+SearchResults.propTypes = {
+  results: PropTypes.arrayOf (
+    PropTypes.object.isRequired,
+  ),
```

Comment:
This really should be a bit more detailed. This is especially important for your app is its a bit more complex than the track specifies and uses more complex data, this would be the perfect place to build out what that should look like